test/suites/tracing-tests/test-trace-call.ts

```diff
@@ -35,5 +35,43 @@ describeSuite({
         expect(logs[1].depth).to.be.equal(1);
       },
     });
+
+    it({
+      id: "T02",
+      title: "should trace contract calls with logs",
+      test: async function () {
+        const contracts = await createContracts(context);
+        const callParams = {
+          to: contracts.callerAddr,
+          data: encodeFunctionData({
+            abi: contracts.abiCaller,
+            functionName: "someAction",
+            args: [contracts.calleeAddr, 6],
+          }),
+        };
+        const tracerParams = {
+          tracerConfig: {
+            withLog: true,
+          },
+        };
+        const traceTx = await customDevRpcRequest("debug_traceCall", [
+          callParams,
+          "latest",
+          tracerParams,
+        ]);
+        const logs: any[] = [];
+        for (const log of traceTx.structLogs) {
+          if (logs.length == 1) {
```

Comment:
Can you explain what is being tested here ? I don't fully understand why the `logs` object is built that way

Reply:
This test was implemented by Alek and was not using the correct tracer now. The test has been updated